packages/react-sdk/docusaurus/docs/React/01-basics/02-introduction.mdx

```diff
@@ -47,7 +38,8 @@ The fastest way to get started with the SDK is by trying the [Video Tutorial](./
 
 We know that everyone consumes documentation differently, which is why we want to make sure everyone can reach their goals as fast as possible.
 
-- Go through it sequentially and look at the next pages (heading to [Installation](./basics/03-installation) next) to get a thorough introduction that covers the most important topics and gets you up to speed in the quickest way.
+- If you haven't yet checked out our [Video Tutorial](./basics/01-tutorial), we suggest starting there.
+- Go through the introducation sequentially and look at the next pages (heading to [SDK architecture](./03-sdk-architecture.mdx) next) to get a thorough introduction that covers the most important topics and gets you up to speed in the quickest way.
 - Search for a specific problem using the search bar at the top, getting guided there directly.
```

Comment:
```suggestion
- search for a specific problem using the search bar at the top
```

Reply:
Maybe we could add a element that when clicked focuses the search input?

Reply:
I'd keep capitalization the same across all bullet points. So either have all as sentences starting with capital letters and punctuation or change all of them.
The wording is something where you know better how to precisely formulate things.